lwrunner/awsrunner.go

```diff
@@ -61,16 +61,17 @@ func NewAWSRunner(amiImageId, host, region, availabilityZone, instanceID string,
 
 	// Heuristically assign SSH username based on AMI name
 	var user string
-	if strings.Contains(*result.Images[0].Name, "ubuntu") {
+	imageName := *result.Images[0].Name
+	if os.Getenv("LW_SSH_USER") != "" {
+		user = os.Getenv("LW_SSH_USER")
+	} else if strings.Contains(imageName, "ubuntu") {
 		user = "ubuntu"
-	} else if strings.Contains(*result.Images[0].Name, "amazon_linux") {
+	} else if strings.Contains(imageName, "amazon_linux") {
+		user = "ec2-user"
+	} else if strings.Contains(imageName, "amzn2-ami") {
 		user = "ec2-user"
```

Comment:
You might consider making this a lookup table, instead of having multiple `else if` statements.

Reply:
How would this work with the `string.Contains` logic? Would we populate the lookup table with all the combinations of strings that will return a hit?

Reply:
You can iterate over the map instead of performing a seek.

Reply:
Done in 81c2c751a662a06174f728c9faee0f519c8a283e